clang-tools-extra/clangd/unittests/tweaks/DefineOutlineTests.cpp

```diff
@@ -349,6 +349,44 @@ TEST_F(DefineOutlineTest, ApplyTest) {
   }
 }
 
+TEST_F(DefineOutlineTest, InCppFile) {
+  FileName = "Test.cpp";
+
+  struct {
+    llvm::StringRef Test;
+    llvm::StringRef ExpectedSource;
+  } Cases[] = {
+      // Member function with some adornments
+      // FIXME: What's with the extra spaces?
+      {
+          "namespace {\n"
```

Comment:
no need for all the complicated test cases, let's just validate the insertion location here for a test case like:
foo.cc
```
namespace foo {
namespace {
struct Foo { void ba^r() {} };
}
}
```